CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 8.1.0
+
+* Fix many library sidebars, and search (#3865)
+* Make getters and setters actually look like their declarations (#3858)
+* Display indirectly 'Implemented types' via extended and mixed-in types (#3855)
+* Add mixins with superclass constraints to the 'Implementers' list. (#3844)
```

Comment:
Nit: some entries are terminated with a `.` while others aren't...

Reply:
I actually saw these too 😄  OK it looks like in other releases we consistently use periods, so I've bumped to do that. Thanks!